packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -91,18 +91,18 @@ export default function ConvertToLinksModal( { onClose, clientId } ) {
 		<Modal
 			closeLabel={ __( 'Close' ) }
 			onRequestClose={ onClose }
-			title={ __( 'Convert to links' ) }
+			title={ __( 'Customize this menu' ) }
 			className={ 'wp-block-page-list-modal' }
 			aria={ { describedby: 'wp-block-page-list-modal__description' } }
 		>
 			<p id={ 'wp-block-page-list-modal__description' }>
 				{ __(
-					'To edit this navigation menu, convert it to single page links. This allows you to add, re-order, remove items, or edit their labels.'
+					'This menu is automatically kept in sync with pages on your site. You can manage the menu yourself by clicking customize below.'
 				) }
 			</p>
 			<p>
 				{ __(
-					"Note: if you add new pages to your site, you'll need to add them to your navigation menu."
+					'Tip: If you want to return to a managed menu, simply add a new menu item and pick the Pages List block.'
```

Comment:
As things stand there doesn't appear to be a way to achieve this. Try it out and you'll see what I mean.

Therefore I don't think we can include this "tip" until we have a new PR which enables this flow.

Reply:
Once this is removed we're good to merge this.